build/helm/edgemesh/charts/agent/values.yaml

```diff
@@ -34,3 +34,7 @@ modules:
   tunnel:
     enable: true
     listenPort: 20006
+    security:
+      enable: false
+      httpServer: ""
```

Comment:
why not also add tlsCAFile, tlsPrivateKeyFile, tlsCertFile?

Reply:
fix